src/validations/test/rules/sar.xspec

```diff
@@ -296,6 +296,330 @@
+                                                            type="tool" />
+                                                    </origin>
+                                                    <subject
+                                                        subject-uuid="75b059f2-a9ba-40b1-a1e0-881196ca1ead"
+                                                        type="component" />
+                                                </observation>
+                                            </result>
+                                        </assessment-results>
+                                    </x:context>
+                                    <x:expect-assert
+                                        id="has-assessment-platform-component"
+                                        label="this is incorrect" />
+                                </x:scenario>
+                                <x:scenario
+                                    label="whose relevant-evidence/@href value matches the ra-tool-output resource in the back-matter">
+                                    <x:context>
+                                        <assessment-results
+                                            xmlns="http://csrc.nist.gov/ns/oscal/1.0">
+                                            <result>
+                                                <observation
+                                                    uuid="6841d8eb-a72c-4672-acc2-2fd265d9617d">
+                                                    <method>TEST</method>
+                                                    <type>finding</type>
+                                                    <subject
+                                                        type="component" />
+                                                    <relevant-evidence
+                                                        href="#19a07333-4e87-46dc-abab-adad60e706b9" />
+                                                </observation>
+                                            </result>
+                                            <back-matter>
+                                                <resource
+                                                    uuid="19a07333-4e87-46dc-abab-adad60e706b9">
+                                                    <prop
+                                                        name="type"
+                                                        value="raw-tool-output" />
+                                                </resource>
+                                            </back-matter>
+                                        </assessment-results>
+                                    </x:context>
+                                    <x:expect-not-assert
+                                        id="has-relevant-evidence"
+                                        label="this is correct" />
+                                </x:scenario>
+                                <x:scenario
```

Comment:
This scenario produces the XSpec report message
>results when a result has an observation with an origin/actor of type = 'tool' that has a subject of type component that has a subject of type component whose relevant-evidence/@href value does not match a the raw-tool-output resource in the back-matter

which is incorrect and redundant.